Web-server crashes in kubernete pod

Error log:
```
module.js:471
    throw err;
    ^
 Error: Cannot find module 'request'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.&lt;anonymous&gt; (/usr/src/app/component/dist/index.js:9:17)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
```
Reason: `Request` package should be `dependencies` now. Otherwise, it won't be copied to docker image.

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -31,6 +31,7 @@
 * Hidden top `go back to old site` for mobile views
 * New Homepage Design
 * Open in National Map button now send config via postMessage (except IE <=11)
+* Fixed web-server crash in in kubernete pod
 
 ## 0.0.36
 

--- a/magda-web-server/package.json
+++ b/magda-web-server/package.json
@@ -35,7 +35,6 @@
         "chai": "^4.1.0",
         "mocha": "^3.4.2",
         "nock": "^9.0.14",
-        "request": "2.85.0",
         "sinon": "^2.4.1",
         "supertest": "^3.0.0",
         "typed-promisify": "^0.4.0",
@@ -51,7 +50,8 @@
         "morgan": "^1.9.0",
         "sitemap": "^1.13.0",
         "urijs": "^1.18.12",
-        "yargs": "^8.0.2"
+        "yargs": "^8.0.2",
+        "request": "2.85.0"
     },
     "config": {
         "docker": {

